exitstats.py

```diff
@@ -131,37 +135,43 @@ def showconn(c):
   t = time.time()
   logf=getlogf(t, snap)
   logf.write("C: %d %s"%(c.cid, time.strftime("%Y-%m-%d-%H:%M:%SZ", time.gmtime(t))))
-  for v in vars:
+  for v in active_vars:
     logf.write(" "+str(snap[v]))
   logf.write("\n")
   logf.flush()
 
 
 # Main
 
-if len(sys.argv) == 1:
-  server=""
-elif len(sys.argv) == 2:
-  server=sys.argv[1]+"/"
-else:
-  print "Usage: %s [server_name]"%sys.argv[0]
-  sys.exit()
-
-a = Web100Agent()
-closed=[]
-while True:
-  cl = a.all_connections()
-  newclosed=[]
-  for c in cl:
-    try:
-      if c.read('State') == 1:
-        newclosed.append(c.cid)
-        if not c.cid in closed:
-          showconn(c)
-    except Exception, e:
-#     print "Exception:", e
-      pass
-  closed=newclosed;
-  time.sleep(5)
-
-
+def main(argv=None):
+  print "Starting exitstats"
+
+  if argv is None:
+    argv = sys.argv
+  if len(argv) == 1:
+      server = ""
+  elif len(argv) == 2:
+    server = argv[1]+"/"
+  else:
+    print "Usage: %s [server_name]"%sys.argv[0]
+    sys.exit()
+
+  a = Web100Agent()
+  closed = []
+  while True:
+    cl = a.all_connections()
+    newclosed = []
+    for c in cl:
+      try:
+        if c.read('State') == 1:
+          newclosed.append(c.cid)
+          if not c.cid in closed:
+            showconn(c)
+      except Exception, e:
+#       print "Exception:", e
+        pass
+    closed=newclosed;
+    time.sleep(5)
+
+if __name__ == "__main__":
+    sys.exit(main())
```

Comment:
Suggest:
```python
sys.exit(main(sys.argv))
```
which allows you to delete 149 and 150
or even dropping the `sys.exit()` and just leaving it as `main(sys.argv)`, because `main` has no return statements.